package.json

```diff
@@ -23,12 +23,6 @@
   "engines": {
     "node": ">=0.10.0"
   },
-  "files": [
-    "app",
-    "dispatcher",
-    "store",
-    "action"
-  ],
```

Comment:
I think back then I was just trying everything out. Normally however I think people use `.npmignore` instead of this.